dotcom-rendering/src/components/NumberedTitleBlockComponent.tsx

```diff
@@ -2,6 +2,7 @@ import { css } from '@emotion/react';
 import { headline } from '@guardian/source-foundations';
 import { decideFormat } from '../lib/decideFormat';
 import { decidePalette } from '../lib/decidePalette';
+import { palette as sourcePalette } from '../palette';
```

Comment:
I don't think this one is the Source palette?

```suggestion
import { palette } from '../palette';
```